ui/src/selectors.js

```diff
@@ -207,6 +219,10 @@ export function selectCollectionMappings(state, collectionId) {
   return selectObject(state, state.collectionMappings, collectionId);
 }
 
+export function selectCollectionDiagrams(state, collectionId) {
+  return selectObject(state.collectionDiagrams, collectionId);
```

Comment:
Why is that an object, does that not hard-code a pagination limit?

Reply:
Good catch, forgot to remove this selector when we changed the api